docs/fastdds/dds_layer/topic/contentFilteredTopic/defaultFilter.rst

```diff
@@ -1,7 +1,358 @@
+  plus or minus sign.
+
+  .. productionlist::
+      FLOATVALUE: (["+"], "-"])? Integer Exponent | (["+"], "-"])? Integer Fractional | | (["+"], "-"])? Integer Fractional Exponent
+      Fractional: "." Integer | "." Integer
+      Exponent: ["e","E"] (["+"], "-"])? Integer
+
+  An example of CHARVALUE:
+
+  .. code::
+
+      value = 10.1e-10
+
+- **STRINGVALUE**: Any series of characters encapsulated in single quotes, except a new-line character or a right quote.
+  A string starts with a left or right quote, but ends with a right quote.
+
+  .. productionlist::
+      STRINGVALUE: ["'"] ~["'", "\r", "\n"] ["'"]
+
+  An example of STRINGVALUE:
+
+  .. code::
+
+      value = 'This is a string'
+
+
+- **ENUMERATEDVALUE**: An enumerated value is a reference to a value declared within an enumeration.
+  Enumerated values consist of the name of the enumeration label enclosed in single quotes.
+  The name used for the enumeration label must correspond to the label names specified in the IDL definition of the
+  enumeration.
+
+  .. productionlist::
+      ENUMERATEDVALUE: ["'"] ~["'", "\r", "\n"] ["'"]
+
+  An example of ENUMERATEDVALUE:
+
+  .. tabs::
+
+      .. tab:: Filter expression
+
+          .. code::
+
+              value = 'ENUM_VALUE_1'
+
+      .. tab:: Associated IDL
+
+          .. code::
+
+              enum MyEnum
+              {
+                  ENUM_VALUE_1,
+                  ENUM_VALUE_2,
+                  ENUM_VALUE_3
+              };
+
+              struct Enumerators
+              {
+                  MyEnum value;
+              };
+
+
+- **PARAMETER**: A parameter is of the form ``%n``, where ``n`` represents a natural number (zero included) smaller than
+  100.
+  It refers to the ``n + 1 th`` argument in the given context.
+
+  .. productionlist::
+      PARAMETER: ["%"] ["0"-"9"] (["0"-"9"])?
+
+  An example of PARAMETER:
+
+  .. code::
+
+      value = %1
+
+
+.. _default_sql_filter_like:
+
+Like condition
+--------------
+
+The ``like`` operator is similar as the one defined by SQL.
+This operator only can be used with strings.
+There are two wildcards could be used in conjunction with this operator
+
+- The percent sign ``%`` (or its alias ``*``) represents zero, one, or multiple characters.
+- The underscore sign ``_`` (or its alias ``?``) represents one single character.
+
+All wildcards can also be used in combinations.
+
+An example of ``like`` operator
+
+.. tabs::
+
+    .. tab:: Filter expression
+
+        .. code::
+
+            "str like '%bird%'"
+
+    .. tab:: Associated IDL
+
+        .. code:: IDL
+
+            struct Like
+            {
+                string str;
+            };
+
+where string "There are birds flying" will be return ``true``.
+
+.. _default_sql_filter_match:
+
+Match condition
+---------------
+
+The ``match`` operator performs a full-text search using a regular expression.
+It uses the Basic Regular Expression (BRE) defined by POSIX.
+More information can be found
```

Comment:
I think we should also indicate that this operator is intended to be used with strings.